src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-setup.mdx

```diff
@@ -0,0 +1,235 @@
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Node.js
+        </td>
+        <td>
+          See the documentation about using the [Node.js agent API](/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api/#external-services).
+        </td>
+      </tr>
+      <tr>
+        <td>
+          PHP
+        </td>
+        <td>
+          See the documentation about using the [PHP agent API](/docs/apm/agents/php-agent/php-agent-api/guide-using-php-agent-api/#datastore) to instrument external calls.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Python
+        </td>
+        <td>
+          No extra steps are necessary: External calls are instrumented automatically.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Ruby
+        </td>
+        <td>
+          See the documentation about using the [Ruby agent API](/docs/apm/agents/ruby-agent/api-guides/guide-using-ruby-agent-api/#externals) to instrument external calls.
+        </td>
+      </tr>
+      </tbody>
+    </table>
+
+## OpenTelemetry [#otel-setup]
+
+The New Relic UI shows external services details once you complete the steps for setting up the OpenTelemetry instrumentation for your service and the [calls](https://opentelemetry.io/docs/java/manual_instrumentation/#span-attributes) it makes to other services. The external services feature shows calls between your services, broken down by transaction endpoints in each service. The name of each transaction is derived from the entry span for the process (`span.kind = "server"`).  
+
+If your APM service is connected to an OpenTelemetry service (upstream or downstream), that OpenTelemetry service will currently not show up in the view for that APM service. This is because, when viewing an APM service, this feature uses metrics which are only reported by APM agents. When viewing an OpenTelemetry service, the APM service will show up as a connection.
+
+The quality of the information you see depends on the sampling strategy you are using in the collector. See the following section about using sampling to control what you see in the UI.
+
+<Callout variant="tip">
+If you send 100% of your OpenTelemetry data to our Trace API, we store 100% of that data, unless you have a specific rate limit for your organization, or if you send enough data to trigger our default rate limit.
+</Callout>
+
+## Adjust sampling to see more UI data [#adjust-sampling]
+
+If you're using an APM agent and are seeing little or no data when you start drilling beyond the initial page of the map, you may need to adjust the span reservoir to sample more data. The reason for this is that transaction-level data is populated by sampled trace data. For OpenTelemetry, both the initial page and all drill-down pages are populated by sampled data, so you may need to make some adjustments in sampling to get the data you need.
```

Comment:
"This is because" rather than "The reason for this is that" ?

Reply:
@barbnewrelic , I shortened this!